gallery/tutorial/micro_gemmini_conv2d.py

```diff
@@ -0,0 +1,221 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Is it necessary to have a tutorial per operator? Wouldn't one tutorial show the general approach?

Reply:
We can have just one example for an entire network, the one for MobileNet, and that would be enough to show that the integration works. But having one per operator allows to understand better the internal workings of the code, and gives some debug capabilities, because one can run specific layers with specific parameters to see if they work or not.